README.md

```diff
@@ -1 +1,52 @@
-# math-magician
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math magicians
+
+> "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+- Make simple calculations.
+- Read a random math-related quote.
+
+## Built With
+
+- React
+
+## Live Demo
+
+[Math Magicians](https://)
+
+## Getting Started
+
+To obtain a local copy of the project follow this steps:
+
+- Open your console.
+- Type the following "git clone https://github.com/diegoyon/math-magician.git" if you are using HTTPS or "git clone git@github.com:diegoyon/math-magician.git" if you are using a SSH key.
+- Press ENTER to create your local clone.
+- Start the SPA by typing `npm start`
+
+## Author
+
+👤 **Diego Yon**
+
+- GitHub: [@diegoyon](https://github.com/diegoyon)
+- Twitter: [@diegoyon](https://twitter.com/diegoyon)
+- LinkedIn: [Diego Yon](https://www.linkedin.com/in/diego-yon-9311aa41/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[OPTIONAL] The MIT link and the Live link in the README file are broken. I would highly recommend fixing those sections to make README professional.